python/tk_framework_desktopserver/shotgun/scripts/cache_commands.py

```diff
@@ -127,12 +128,12 @@ def cache(cache_file, data, base_configuration, engine_name, config_data, config
         ])
     else:
         if config_is_mutable:
-            logger.debug("Not a Project entity: not registering core and app update commands.")
+            engine.log_debug("Not a Project entity: not registering core and app update commands.")
```

Comment:
it seems a little odd that we are testing for `if config_is_mutable` but then the debug message says "not a project entity:"